tests/components/laundrify/test_sensor.py

```diff
@@ -0,0 +1,104 @@
+"""Test the laundrify sensor platform."""
+
+from datetime import timedelta
+import json
+import logging
+from unittest.mock import AsyncMock, patch
+
+from laundrify_aio import LaundrifyDevice
+import pytest
+
+from homeassistant.components.laundrify.const import DOMAIN, MANUFACTURER, MODELS
+from homeassistant.components.laundrify.sensor import LaundrifyPowerSensor
+from homeassistant.components.sensor import SensorDeviceClass
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    ATTR_UNIT_OF_MEASUREMENT,
+    STATE_UNKNOWN,
+    UnitOfPower,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+from homeassistant.util import slugify
+from homeassistant.util.dt import utcnow
+
+from tests.common import MockConfigEntry, async_fire_time_changed, load_fixture
+
+
+@pytest.fixture(name="laundrify_sensor")
+def laundrify_sensor_fixture() -> LaundrifyPowerSensor:
+    """Return a default Laundrify power sensor mock."""
+    # Load test data from machines.json
+    machine_data = json.loads(load_fixture("laundrify/machines.json"))[0]
+
+    mock_device = AsyncMock(spec=LaundrifyDevice)
+    mock_device.id = machine_data["id"]
+    mock_device.manufacturer = MANUFACTURER
+    mock_device.model = machine_data["model"]
+    mock_device.name = machine_data["name"]
+    mock_device.firmwareVersion = machine_data["firmwareVersion"]
+    return LaundrifyPowerSensor(mock_device)
```

Comment:
We should not use the entities directly. We should mock the library with the data we expect and then check the state machine if everything went well